apps/web/src/components/card-pull-request/timeline-pull-request.tsx

```
@@ -0,0 +1,187 @@
+        }
+        title={hasReviews ? "First reviewed" : "First review"}
+        c={getColor(2)}
+      >
+        <Text size="xs" mt={4}>
+          {hasReviews && timeToFirstReview && (
+            <>in {humanizeDuration(timeToFirstReview)}</>
+          )}
+          {!hasReviews && timeToFirstReview && (
+            <>awaiting for {humanizeDuration(timeToFirstReview)}</>
+          )}
+          {!hasReviews && isDone && <>Skipped</>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)